### PR TITLE
[WIP] events timeline

### PR DESCRIPTION
**Pull request by jasonLaster**
image

image

### CODE REVIEW

File: src/ui/components/EventsTimeline.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.events-timeline {
+  width: 300px;
+  height: 100%;
+  /* border: 2px solid rebeccapurple; */
```
Comment (princiya) [inline src/ui/components/EventsTimeline.css:4]: wow! I din't know `rebeccapurple` was a color 😮

File: src/ui/components/EventsTimeline.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+import "./EventsTimeline.css";
+
+export default function EventsTimeline() {
+  return (
+    <div className="events-timeline">
+      <div>
+        <div className="event ">
+          <div className="img rxjs"></div>
+          <div className="label">I recorded a quick todo list with 3 completed todos</div>
```
Comment (princiya) [inline src/ui/components/EventsTimeline.js:12]: How's this meant to get populated or be dynamic?

Comment (jasonLaster) [reply src/ui/components/EventsTimeline.js:12]: :) yeah, we have to pull the data from somewhere

**Comment by jasonLaster** [comment]
Screen Shot 2020-08-16 at 8 32 55 AM

**Pull request closed without merging**